src/js/elements/menu.js

```diff
@@ -95,13 +96,15 @@ export function loadMenu(dataMapperMenu, data, subindicatorCallback) {
         $(".data-category__h3", h3Wrapper).remove();
 
         for (const [indicator, detail] of Object.entries(indicators)) {
-            let newIndicator = indicatorClone.cloneNode(true);
-            $('.truncate', newIndicator).text(indicator);
-            $(h3Wrapper).append(newIndicator);
-            const childWrapper = $(newIndicator).find('.data-category__h3_wrapper');
-
-            let subindicators = detail.metadata.groups.filter((group) => group.name === detail.metadata.primary_group)[0];
-            addSubIndicators(childWrapper, category, subcategory, indicator, subindicators, indicators, detail);
+            if (detail.content_type != ContentBlock.BLOCK_TYPES.HTMLBlock) {
```

Comment:
I think it would be better to use dataset_content_type rather than content_type
![Screenshot from 2021-08-17 11-05-05](https://user-images.githubusercontent.com/420786/129697225-0bc5bd48-857e-462a-ac11-6082ddd8af64.png)